Tools/AP_Periph/Parameters.cpp

```diff
@@ -563,6 +563,16 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     GOBJECT(rpm_sensor, "RPM", AP_RPM),
 #endif
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    // @Group: SIM_
+    // @Path: ../libraries/SITL/SITL.cpp
+    GOBJECT(sitl, "SIM_", SITL::SIM),
+
+    // @Group: AHRS_
```

Comment:
```suggestion
#endif
#if AP_AHRS_ENABLED
    // @Group: AHRS_
```

... similar changes elsewhere.  Will needa line in boards.py enabling it.

Reply:
done